### PR TITLE
Appropriate seperator added to different locales

### LINKED ISSUE #1526: Comma and dot are used in no particular order when the device locale is set to language with comma separator
**Actual Behaviour**

On the same screen, there can be numbers with dot and comma separators.  

**Expected Behaviour**

If a comma is used in locale, then all numbers are shown with a comma

**Steps to reproduce it**

Set phone language to Russian for example

**Screenshots of the issue**

image

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by ho-dor**
Fixes #1526 

**Changes**: A lot of places where dot separator was used instead of comma separator in local languages are fixed. 

**Screenshot/s for the changes**: 
issue

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug (2).zip](https://github.com/fossasia/pslab-android/files/2947435/app-debug.2.zip)

### CODE REVIEW

**Comment by ho-dor** [comment]
@CloudyPadmal @abhinavraj23 please review

**Comment by mariobehling** [comment]
The comma or dot should depend on the locale. In English locales the formatting is usually a dot while in some other formats it could be a dot. Please research and implement it to show the right formatting according to a language.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1091,7 +1082,10 @@
     GND. GND is meant for Ground and any of the PSLab device GND pins can be used since they are common.\n\n
     \u2022 Select sensor by going to the Configure tab from the bottom navigation bar and choose BHT-1750 in the drop down menu under Select Sensor.\n</string>
     <string name="no_data_fetched">No Data Fetched</string>
-    <string name="recorder">Recorder</string>
     <string name="save_graph">Save Graph</string>
 
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (cweitat) [inline app/src/main/res/values/strings.xml]: what is this for?

**Comment by ho-dor** [comment]
@mariobehling I have not explicitly used commas or dots . I have used DataFormatter which adapts according to the locale.

**Comment by ho-dor** [comment]
@mariobehling please review this .

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -153,6 +153,7 @@
     <string name="accelerometer_intro">Accelerometer is used to measure acceleration of a body along the X, Y and Z axis.\n</string>
     <string name="accelerometer_description_text">\n\nThe figure above shows the direction of all the three axis when the mobile is held straight.\n\nSteps to measure acceleration in PSLab app:\n\n\u2022 Hold the device as shown in the above figure.\n\n\u2022 Accelerate the device along any one or multiple axis.\n\n\u2022 Observe the values in the cards or the plotted graph of any particular axis.\n\nThe Accelerometer instrument can also be used to measure the acceleration of a moving body by placing the device on/inside the body and then accelerating the body.\n\nNOTE: Don\'t accelerate the body if the device isn\'t properly attached else the device could be damaged. </string>
     <string name="accelerometer_configurations">Accelerometer Configurations</string>
+    <string name="no_accelerometer_sensor">Device does not have a Accelerometer sensor</string>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: Device does not have any accelerometer sensor

**Comment by ho-dor** [comment]
@CloudyPadmal @cweitat please review

**Comment by ho-dor** [comment]
This has been pending for some time @abhinavraj23 @CloudyPadmal @cweitat

**Comment by cweitat** [comment]
@ho-dor please see the conflict file.

**Comment by ho-dor** [comment]
@cweitat please review

**Pull request merged**